input/pagecontent/use-cases.md

```diff
@@ -1,10 +1,8 @@
-In the context of AU eRequesting, use cases focus on standirdising clinical requesting and ordering within the Australian healthcare system, specifically targeting Pathology and Imaging requests in AU eRequesting Release 1 (R1). The use cases aim to ensure that the system supports accurate and efficient data exchange and workflow management.
+The use cases for AU eRequesting focus on standirdising clinical requesting and ordering within the Australian healthcare system, specifically targeting Pathology and Imaging requests in AU eRequesting R1. These use cases aim to ensure systems support accurate and efficient data exchange and workflow management.
```

Comment:
Replace this text with:
The use cases below are those in scope of the first release of AU eRequesting. These use cases are scoped to pathology and medical imaging requesting in community-based care provision

Reply:
done